modules/mobile/spec/request/appointments_list_request_spec.rb

```diff
@@ -91,35 +91,45 @@
     end
 
     context 'with valid params when there are cached appointments' do
-      let(:start_date) { Time.now.utc.iso8601 }
+      va_path = Rails.root.join('modules', 'mobile', 'spec', 'support', 'fixtures', 'va_appointments.json')
+      cc_path = Rails.root.join('modules', 'mobile', 'spec', 'support', 'fixtures', 'cc_appointments.json')
+      va_json = File.read(va_path)
+      cc_json = File.read(cc_path)
+      va_appointments = Mobile::V0::Adapters::VAAppointments.new.parse(
+        JSON.parse(va_json, symbolize_names: true)
+      )
+      cc_appointments = Mobile::V0::Adapters::CommunityCareAppointments.new.parse(
+        JSON.parse(cc_json, symbolize_names: true)
+      )
+
+      appointments = (va_appointments + cc_appointments).sort_by(&:start_date_utc)
+
+      let(:start_date) { (Time.now.utc.beginning_of_year - 1.year).iso8601 }
```

Comment:
Update caching specs to be at beginning of previous year but this is just implicitly testing the start date limit. Let me know if you think this should be explicitly tested.

Reply:
I asked myself the same question when I was working on appointment requests and I came to the same conclusion that the start date is being implicitly tested by VCR. However, given that the change here is specific to the date ranges and (AFAICT from a quick search through this file) we don't seem to explicitly test the default date setting behavior, it might be worth a more explicit testing approach. Maybe something like:
```
context 'when start date is after the beginning of the previous year' do
  it 'defaults start date to the beginning of the previous year' do
    expect(Faraday).to have_received(:get).with(url, beginning_of_year)
  end
end
```
That's an oversimplification, but you get the idea. A slightly easier way would be to do the same with the proxy, but this is a less valuable test because these are integration tests that also test the proxy, which does not have its own unit tests. 
```
expect_any_instance_of(Mobile::V0::Appointments::Proxy).to receive(:get_appointments).with(start_date: beginning_of_last_year)
```
I'm happy to pair with you on this.